.github/workflows/gpu_tests.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           git config --global user.email "username@users.noreply.github.com"
           echo "${GITHUB_REPOSITORY}" > repo
           echo "${{ inputs.SHA || github.sha }}" > commit
-          git commit -m "Test commit ${GITHUB_SHA}" repo commit
+          git commit -m "Test commit ${{ inputs.SHA || github.sha }}" repo commit
```

Comment:
Are `GITHUB_SHA` and `github.sha` the same ?

Reply:
Yes they should be. `github.sha` is calculated by github when it runs the commands `GITHUB_SHA` is an environment variable